source/index.html.md

```diff
@@ -287,12 +287,13 @@ You should build tests on your `post_raw_data` field to check numbers before you
 
 ## Extra Helpful Info
 <aside class="notice">
+<br>
 Posting summary information can be disorienting when it appears as a journal entry with no context. So to help with this situation, we need to add context.  The places to do this are in the `je_private_note` the `source_report_url` and in any node filed with `_note` attached.  In addition, adding an attachment of a csv file with all transactions summarized is extremely helpful. 
-
+<br>
 Examples of extra helpful context are:
 The url of a report specifically for the date period of the summary and the currency and time zone so that it matches exactly to the summary.  That url should go into `source_report_url` and also into `je_private_note`. 
 Adding a `_note` to any node will add that text next to the exact number.  A great place to use this is for deposits and expected deposits.  An example of a helpful entry is below. 
-
+<br>
```

Comment:
Not sure if this will work, it might, I don't know what Slate does in its transliteration (it doesn't work in straight markdown). We could throw it over the wall and see if it works and if it doesn't try this:

"When you do want to insert a break tag using Markdown, you end a line with two or more spaces, then type return."